src/presence_analyzer/views.py

```diff
@@ -75,6 +86,55 @@ def users_view():
     ]
 
 
+@app.route('/api/v1/months', methods=['GET'])
+@jsonify
+def months_view():
+    """
+    Years/months listing for dropdown.
+    """
+    return get_dates()
+
+
+@app.route('/api/v1/top_5/<string:month_year>', methods=['GET'])
+@jsonify
+def top_5(month_year):
+    """
+    Returns top 5 employees of given month.
+    """
+    data = get_data()
+    top_presence = OrderedDict()
+
+    try:
+        month, year = month_year.split('-')
+    except (IndexError, ValueError):
+        log.debug('%s is not a correct format!', month_year)
+        abort(404)
+
+    if len(year) < 4 or not year.isdigit() or month not in month_name:
+        log.debug('%s is not a correct format!', month_year)
+        abort(404)
+
+    top_presence = {
+        user: {
+            'total_hours': total_hours(data[user]['presence'], month, year),
+            'image': prepare_photo_url(user).data[1:-1],
+            'name': data[user]['name']
+        } for user in data
+    }
+
+    top5 = list(OrderedDict(
+        sorted(
+            top_presence.items(),
+            key=lambda x: x[1]['total_hours'],
+            reverse=True
+        )
+    ).items())[:5]
+    if not top5[0][1]['total_hours']:
+        log.debug('No data for year %s', year)
+        abort(404)
+    return top5
```

Comment:
Imo there's to much logic here. Such operations should be in some kind of util functions or so. Views should be as light as possible.